cmd/asset-syncer/utils_test.go

```diff
@@ -581,20 +582,7 @@ func Test_fetchAndImportFiles(t *testing.T) {
 		m := mock.Mock{}
 		m.On("One", mock.Anything).Return(errors.New("return an error when checking if files already exists to force fetching"))
 		chartFilesID := fmt.Sprintf("%s/%s-%s", charts[0].Repo.Name, charts[0].Name, cv.Version)
-		m.On("UpsertId", chartFilesID, chartFiles{chartFilesID, testChartReadme, testChartValues, testChartSchema, charts[0].Repo, cv.Digest})
-		manager := getMockManager(&m)
-		fImporter := fileImporter{manager}
-		err := fImporter.fetchAndImportFiles(charts[0].Name, charts[0].Repo, cv)
-		assert.NoErr(t, err)
-		m.AssertExpectations(t)
-	})
-
-	t.Run("valid tarball", func(t *testing.T) {
-		netClient = &goodTarballClient{c: charts[0]}
-		m := mock.Mock{}
-		m.On("One", mock.Anything).Return(errors.New("return an error when checking if files already exists to force fetching"))
-		chartFilesID := fmt.Sprintf("%s/%s-%s", charts[0].Repo.Name, charts[0].Name, cv.Version)
-		m.On("UpsertId", chartFilesID, chartFiles{chartFilesID, testChartReadme, testChartValues, testChartSchema, charts[0].Repo, cv.Digest})
```

Comment:
Why this removal of a test?

Reply:
completely overlooked this one, this test doesn't have to be removed :/